CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ For older changes see the [archived Singularity change log](https://github.com/a
 
 ## Changes for v1.3.x
 
+- Fixed the time discrepancy issue between the host and container, which occurs
+  when host's timezone is different from container's.
```

Comment:
This isn't quite accurate.  I suggest instead.
```
- Stopped binding over the default timezone in the container with the host's timezone, which led to unexpected behavior if the application changed timezones.
```

Reply:
thanks! updated in the latest commit.